index.html

```diff
@@ -1,176 +1,780 @@
-
-                    <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-                        <input class="mdl-textfield__input" type="email" value="" name="EMAIL" id="mce-EMAIL"
-                            data-required>
-                        <label class="mdl-textfield__label" for="mce-EMAIL">Email address</label>
-                    </div>
 
-                    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-                    <div style="position: absolute; left: -5000px;" aria-hidden="true">
-                        <input type="text" name="b_84b372b9598d22040af9583e0_0def27f7e4" tabindex="-1" value="">
-                    </div>
+  <!-- ***** Preloader Start ***** -->
+  <div id="js-preloader" class="js-preloader">
+    <div class="preloader-inner">
+      <span class="dot"></span>
+      <div class="dots">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
+    </div>
+  </div>
+  <!-- ***** Preloader End ***** -->
 
-                    <button type="submit" name="subscribe" id="mc-embedded-subscribe"
-                        class="mdl-button mdl-button--raised mdl-button--primary">
-                        Subscribe
-                    </button>
-            </div>
-            </form>
-        </div>
+  <!-- ***** Header Area Start ***** -->
+  <header class="header-area header-sticky wow slideInDown" data-wow-duration="0.75s" data-wow-delay="0s">
+    <div class="container">
+      <div class="row">
+        <div class="col-12">
+          <nav class="main-nav">
+            <!-- ***** Logo Start ***** -->
+            <a href="index.html" class="logo">
+              <img src="assets/images/qe/quanteco2.png" alt="Quanteco" style="width: 130px;">
+            </a>
+            <!-- ***** Logo End ***** -->
+            <!-- ***** Menu Start ***** -->
+            <ul class="nav">
+              <li class="scroll-to-section"><a href="#top" class="active">Login</a></li>
+              <li class="scroll-to-section"><a href="#services">Foodprint</a></li>
+              <li class="scroll-to-section"><a href="#about">About us</a></li>
+             <!-- <li class="scroll-to-section"><a href="#pricing">Partnerships</a></li>-->
+              <li class="scroll-to-section"><a href="#newsletter">Newsletter</a></li>
+              <li><div class="gradient-button"><a id="modal_trigger" href="https://staging.quanteco.org" target="_blank" ><!--   href="#modal"  > <i class="fa fa-sign-in-alt"></i>-->To the alpha!</a></div></li> 
+            </ul>        
+            <a class='menu-trigger'>
+                <span>Menu</span>
+            </a>
+            <!-- ***** Menu End ***** -->
+          </nav>
         </div>
+      </div>
+    </div>
+  </header>
+  <!-- ***** Header Area End ***** -->
+  
+  <div id="modal" class="popupContainer" style="display:none;">
```

Comment:
I feel like this whole `div` is just for login and can therefore be removed?

Reply:
Just keeping it for if and when it's useful. Was planning to leave these redundancies there for a few months until we have actual users